wordpress-mobile/WordPress-iOS
# [Dark Mode] Multiple Fixes

Refs. #12320 

This PR fixes:
- Muriel placeholder color: [bd29ac8](https://github.com/wordpress-mobile/WordPress-iOS/commit/bd29ac834e1ace71c5838c02ee47bc71df49ec24)

- Page List cell labels colors & Delete Button in Site Settings
![dark-mode-mix](https://user-images.githubusercontent.com/912252/64436471-52050300-d0bc-11e9-8fdf-d3b3e2e2c510.jpg)
- Site creation flow
![site-creation-dark-mode-dark-01](https://user-images.githubusercontent.com/912252/64437926-13bd1300-d0bf-11e9-9a68-d6073afa48ee.jpg)
![site-creation-dark-mode-dark-02](https://user-images.githubusercontent.com/912252/64437927-13bd1300-d0bf-11e9-9947-c436bbdad3f4.jpg)
![site-creation-dark-mode-dark-03](https://user-images.githubusercontent.com/912252/64437928-13bd1300-d0bf-11e9-9178-dcc3e0c6e866.jpg)
![site-creation-dark-mode-dark-04](https://user-images.githubusercontent.com/912252/64437929-13bd1300-d0bf-11e9-947e-75f6577f2c81.jpg)
![site-creation-dark-mode-light-01](https://user-images.githubusercontent.com/912252/64438847-b7f38980-d0c0-11e9-9732-cac849da339b.jpg)
![site-creation-dark-mode-light-02](https://user-images.githubusercontent.com/912252/64438848-b88c2000-d0c0-11e9-87ef-ebeef7cb7eba.jpg)
![site-creation-dark-mode-light-03](https://user-images.githubusercontent.com/912252/64438849-b88c2000-d0c0-11e9-858c-17922d5adfef.jpg)
![site-creation-dark-mode-light-04](https://user-images.githubusercontent.com/912252/64438852-b88c2000-d0c0-11e9-9bc2-fe847abf5187.jpg)
- Fix Quick Start
![quick-start-dark-mode-dark](https://user-images.githubusercontent.com/912252/64437193-c42a1780-d0bd-11e9-9000-e27d4cde84e5.jpg)
![quick-start-dark-mode-light](https://user-images.githubusercontent.com/912252/64437210-ce4c1600-d0bd-11e9-8440-fc10f56c7bcb.jpg)

## To test:
• Run this branch with Xcode 11 and iOS 13
• Create a new WordPress.com site and check everything works fine in dark and light mode
• Then activate QuickStart and check the site dashboard header has the right color.
• Check the 2 QuickStart checklist have the right colors
• Go to the site settings and try to cancel it. Check the cancel button has the right color.
• Do the same thing building for iOS 12

Update release notes:
